torch/testing/_internal/common_cuda.py

```diff
@@ -27,8 +27,9 @@
     TEST_MAGMA = torch.cuda.has_magma
 
 if TEST_NUMBA:
-    import numba.cuda
-    TEST_NUMBA_CUDA = numba.cuda.is_available()
+    # import numba.cuda
```

Comment:
This needs to be reverted before merging.

Reply:
oops